PyDSS/pyPlots/Plots/NetworkGraph.py

```diff
@@ -1,33 +1,37 @@
 import pandas as pd
 import networkx as nx
 import os
-
 from bokeh.client import push_session
 from bokeh.io import show, output_file
 
 from bokeh.plotting import figure, curdoc
 from bokeh.models import ColumnDataSource, HoverTool, BoxSelectTool, BoxZoomTool, \
     PanTool, WheelZoomTool, ResetTool, SaveTool
-
+from  PyDSS.pyPlots.pyPlotAbstract import PlotAbstract
 import matplotlib.colors as colors
+import opendssdirect as dss
 
+class NetworkGraph(PlotAbstract):
 
-class CreateGraph:
-
-    def __init__(self, dssInstance):
-        self.__dssInstance = dssInstance
-        self.__dssGraph = nx.DiGraph()
+    def __init__(self, PlotProperties, dssBuses, dssObjectsbyClass, dssCircuit, dssSolver):
+        super(NetworkGraph).__init__()
+        print(PlotProperties)
+        self._settings = PlotProperties
+        self.__dssInstance = dss
+        a = {}
+        self.__dssGraph = nx.DiGraph(**a)
```

Comment:
Since `a` is empty could you use this instead?
```
self.__dssGraph = nx.DiGraph()
```

Reply:
Was testing a few hings with the new version of NX. missed when I was removing changes